internal/service/sagemaker/sagemaker_test.go

```diff
@@ -46,6 +46,29 @@ func TestAccSageMaker_serial(t *testing.T) {
 			"kernelGatewayAppSettings":                 testAccDomain_kernelGatewayAppSettings,
 			"kernelGatewayAppSettings_customImage":     testAccDomain_kernelGatewayAppSettings_customImage,
 			"kernelGatewayAppSettings_lifecycleConfig": testAccDomain_kernelGatewayAppSettings_lifecycleConfig,
+			"kernelGatewayAppSettings_defaultResourceAndCustomImage": testAccDomain_kernelGatewayAppSettings_defaultResourceSpecAndCustomImage,
```

Comment:
something looks incorrect here with this list addition, it should be one addition and one rename

Reply:
Sorry, fixed now.